strapi/.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[{package.json,*.yml}]
+indent_style = space
+indent_size = 2
```

Comment:
See above
```suggestion
indent_size = 4
```

Reply:
This is for the strapi editor right ? It's not clear to me where this will stand ? Who will use it/have to deal with these indents ?

Reply:
No it's a set of conventions for text editors in general (see https://editorconfig.org/) so code maintainers may use this setting. In this case, this will apply to all `package.json` and `.yaml` files inside the `strapi/` folder, when you open it with some text editor. A bunch of IDE support it out of the box, but we will enforce linting rules on TS code anyway. However for those config files, we could modify the indent value.

Reply:
All right let's keep it as this so I guess